pkg/types/apisix/v1/types.go

```diff
@@ -79,16 +79,37 @@ type Metadata struct {
 type Route struct {
 	Metadata `json:",inline" yaml:",inline"`
 
-	Host        string            `json:"host,omitempty" yaml:"host,omitempty"`
-	Hosts       []string          `json:"hosts,omitempty" yaml:"hosts,omitempty"`
-	Uri         string            `json:"uri,omitempty" yaml:"uri,omitempty"`
-	Priority    int               `json:"priority,omitempty" yaml:"priority,omitempty"`
-	Vars        [][]StringOrSlice `json:"vars,omitempty" yaml:"vars,omitempty"`
-	Uris        []string          `json:"uris,omitempty" yaml:"uris,omitempty"`
-	Methods     []string          `json:"methods,omitempty" yaml:"methods,omitempty"`
-	RemoteAddrs []string          `json:"remote_addrs,omitempty" yaml:"remote_addrs,omitempty"`
-	UpstreamId  string            `json:"upstream_id,omitempty" yaml:"upstream_id,omitempty"`
-	Plugins     Plugins           `json:"plugins,omitempty" yaml:"plugins,omitempty"`
+	Host        string   `json:"host,omitempty" yaml:"host,omitempty"`
+	Hosts       []string `json:"hosts,omitempty" yaml:"hosts,omitempty"`
+	Uri         string   `json:"uri,omitempty" yaml:"uri,omitempty"`
+	Priority    int      `json:"priority,omitempty" yaml:"priority,omitempty"`
+	Vars        Vars     `json:"vars,omitempty" yaml:"vars,omitempty"`
+	Uris        []string `json:"uris,omitempty" yaml:"uris,omitempty"`
+	Methods     []string `json:"methods,omitempty" yaml:"methods,omitempty"`
+	RemoteAddrs []string `json:"remote_addrs,omitempty" yaml:"remote_addrs,omitempty"`
+	UpstreamId  string   `json:"upstream_id,omitempty" yaml:"upstream_id,omitempty"`
+	Plugins     Plugins  `json:"plugins,omitempty" yaml:"plugins,omitempty"`
+}
+
```

Comment:
Add a comment for it.

```suggestion
// Vars represents the route match expressions of APISIX.
type Vars [][]StringOrSlice
```

Reply:
The comment for Vars type declare has been added.